torch_geometric/loader/utils.py

```diff
@@ -91,7 +95,7 @@ def filter_edge_store_(store: EdgeStorage, out_store: EdgeStorage, row: Tensor,
             edge_attr = value.storage.value()
             if edge_attr is not None:
                 index = index.to(edge_attr.device)
-                edge_attr = edge_attr[index]
+                edge_attr = index_select(edge_attr, index, dim=0)
```

Comment:
If we already guaranteed int64, do we still need to change here?

Reply:
I don't think the `index` is guaranteed to be `int64` here.